Client/src/Components/GroupedTask/GroupedTask.jsx

```diff
@@ -160,8 +144,8 @@ export default function GroupedTask(props){
           {selectAll && <input type="checkbox"/>}
           <p className = "left" >{dateStr}</p>
           <div className = "right">
-            <p>Total Time: {convertElapsedToText(timeTotals[index])}</p>
```

Comment:
This change also didn't seem to make it through your merge.  Could you do the same here, copy and paste the deleted version in red and replace the unrevised version in your file before making a new commit and pull request?

Reply:
copied & pasted deleted version in red